src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -42,12 +43,14 @@
  * running on Kafka Connect Workers.
  */
 public class SnowflakeSinkConnector extends SinkConnector {
+  // TEMPORARY config of num tasks assigned per client, round up if number is not divisible
+  private static final int NUM_TASK_TO_CLIENT = 1;
```

Comment:
Since this is final, does this mean we will have each task create its own client?

Reply:
yes, this is just to preserve the current functionality so that I can complete this PR. I'll do further perf testing to see if there is any differences in throughput or memory usage using 1:1 or 1:many ratio of client to task

Reply:
Got it, can you mention in the description that this is just a refactor to support 1:many in future and there is no behavior change at least for this change.

Reply:
added in description and an additional comment in the file